### PR TITLE
[NNVM][DARKNET]Logistic activation added

### PR DESCRIPTION
**Pull request by siju-samuel**
Logistic activation support is added for darknet, which is needed for a C-RNN  model.

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from others in the community.

### CODE REVIEW

File: nnvm/python/nnvm/frontend/darknet.py [hunk 1]
```diff
@@ -323,6 +325,8 @@ def _darknet_activations(inputs, attrs):
         sym = _darknet_get_nnvm_op(op_name)(*inputs, **new_attrs)
     elif act_type in ['elu']:
         sym = -1 * _sym.relu(1 - _sym.exp(*inputs)) + _sym.relu(*inputs)
+    elif act_type in ['logistic']:
+        sym = 1 / (1 + _sym.exp(*inputs * (-1)))
```
Comment (PariksheetPinjari909) [inline nnvm/python/nnvm/frontend/darknet.py]: _sym.sigmoid function can be used here.

File: nnvm/tests/python/frontend/darknet/test_forward.py [hunk 2]
```diff
@@ -324,6 +324,32 @@ def test_forward_rnn():
     test_rnn_forward(net)
     LIB.free_network(net)
 
+def test_forward_activation_logistic():
+    '''test elu activation layer'''
```
Comment (PariksheetPinjari909) [inline nnvm/tests/python/frontend/darknet/test_forward.py]: Pls correct comment elu to logistic activation

**Comment by siju-samuel** [comment]
@PariksheetPinjari909 Thanks for reviewing. I modified according to your comments. Please check.

**Comment by tqchen** [comment]
Thanks @siju-samuel @PariksheetPinjari909 ! this is now merged

**Pull request merged**